src/Resources/RecipeApiCall.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Mamoi\Resources;
+
+class RecipeApiCall {
+
+    /**Initialises curl Request and fetches an assoc array from the http://www.recipepuppy.com API
+     *
+     * @param string $url which is created when user selects and submits items from then ingredients list
+     *
+     * @return array of results from the recipepuppy API with a list of meals
+     */
+    public static function createCurl(string $query) : array
+    {
+        $url = 'http://www.recipepuppy.com/api/?i=' . $query;
+        $curlRequest = curl_init();
+        curl_setopt($curlRequest, CURLOPT_URL, $url);
+        curl_setopt($curlRequest, CURLOPT_RETURNTRANSFER, 1);
+        $output = curl_exec($curlRequest);
+        $result = json_decode($output, true);
+        curl_close($curlRequest);
+        if(array_key_exists('results', $result)) {
```

Comment:
space after if

Reply:
I like the idea of checking for `results` in the response, but what happens if the API goes down entirely? You won't get an array back from the request so passing it into `array_key_exists()` here will trigger a warning. It's not a huge issue as it's quite niche, and warnings will generally be suppressed on production servers so no one would see it, but best to account for it if you can.